### PR TITLE
[a11y] improvements

### PR DESCRIPTION
**Pull request by d1nocan**
I add some accessibility improvements.

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const Copyright = () => {
   return (
-    <p className="text-center text-neutral-400 dark:text-neutral-600">
+    <p className="text-center text-neutral-700 dark:text-neutral-400">
```
Comment (xilaluna) [inline src/components/Footer.tsx]: I made the text specifically to be subtle that's why they were not bright. Could you revert your changes to the original. thx

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const Copyright = () => {
 
 const Footer: React.FC = () => {
   return (
-    <footer className="border-color container mt-auto border-t py-3 sm:py-5">
```
Comment (xilaluna) [inline src/components/Footer.tsx]: could you revert your changes here as well there was nothing changed except the ordering of the classes

File: src/components/Navbar.tsx [hunk 3]
```diff
@@ -77,22 +77,22 @@ const Navbar = () => {
             }`}
           />
         </Link>
-        <Link href={'/skills'}>
+        <Link href={'/skills'} aria-label="Skils">
```
Comment (xilaluna) [inline src/components/Navbar.tsx]:
```suggestion
        <Link href={'/skills'} aria-label="Skills">
```

File: src/styles/globals.css [hunk 4]
```diff
@@ -25,28 +25,28 @@
...
+    @apply container py-10 space-y-8 sm:space-y-10 sm:py-20;
   }
   .btn {
-    @apply rounded-md bg-pink-400 px-4 py-2 text-center text-neutral-200 transition duration-300 ease-in-out hover:bg-neutral-900 dark:bg-indigo-400 dark:text-neutral-800 dark:hover:bg-neutral-100;
+    @apply px-4 py-2 text-center transition duration-300 ease-in-out bg-pink-400 rounded-md text-neutral-50 hover:bg-neutral-900 dark:bg-indigo-400 dark:text-neutral-800 dark:hover:bg-neutral-100;
   }
   .secondary-btn {
-    @apply rounded-md border border-pink-400 px-4 py-2 text-pink-400 transition duration-300 ease-in-out hover:border-pink-500 hover:bg-pink-300/25 dark:border-indigo-400 dark:text-indigo-400 dark:hover:border-indigo-500 dark:hover:bg-indigo-900/25;
+    @apply px-4 py-2 text-pink-400 transition duration-300 ease-in-out border border-pink-400 rounded-md hover:border-pink-500 hover:bg-pink-300/25 dark:border-indigo-400 dark:text-indigo-400 dark:hover:border-indigo-500 dark:hover:bg-indigo-900/25;
   }
```
Comment (xilaluna) [inline src/styles/globals.css]: revert changes

**Review by xilaluna — CHANGES_REQUESTED** [review]
Thank you for your pull request and your contribution, there are only a few changes I ask of.

File: src/components/Footer.tsx (hunk 1 above)
Comment (d1nocan) [reply src/components/Footer.tsx]: I liked the colors of the site and I don't want to change them, but some colors reduce readability.

File: src/components/Footer.tsx (hunk 2 above)
Comment (d1nocan) [reply src/components/Footer.tsx]: Sorry, the extension I used changed the ordering automatically

**Comment by d1nocan** [comment]
I'm sorry I made a few changes to the design. It will be better if I write what can be changed from here rather than making changes to the code.

Your dark theme is almost perfect, but I can't say that for your light theme.

Your color tones chooses can make some texts unreadable, especially on a small screen.

Although I don't use the light theme, I think it doesn't look nice from the point of view of those who use it.

I hope your site will become even more beautiful in the future. By the way, I liked your projects on the site.

**Review by xilaluna — APPROVED** [review]
Thank you so much, I really appreciate your additions.

**Pull request merged**